proto/xion/v1/query.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package xion.v1;
+
+option go_package = "github.com/burnt-labs/xion/x/xion/types";
+
+service Query {
+  rpc WebAuthNVerifyRegister(QueryWebAuthNVerifyRegisterRequest) returns (QueryWebAuthNVerifyRegisterResponse) {}
+  rpc WebAuthNVerifyAuthenticate(QueryWebAuthNVerifyAuthenticateRequest) returns (QueryWebAuthNVerifyAuthenticateResponse) {}
+}
+
+message QueryWebAuthNVerifyRegisterRequest {
+  string addr = 1;
+  string challenge = 2;
+  string rp = 3;
+  bytes data = 4;
+}
+
+message QueryWebAuthNVerifyRegisterResponse {
+  bytes credential = 1;
+}
+
+message QueryWebAuthNVerifyAuthenticateRequest {
+  string addr = 1;
+  string challenge = 2;
+  string rp = 3;
+  bytes credential = 4;
+  bytes data = 5;
+}
+
+message QueryWebAuthNVerifyAuthenticateResponse {}
```

Comment:
I'm not sure I quite get the use for this Query? given that it's empty? It's being used: https://github.com/burnt-labs/xion/blob/0656a495bd6e66a86b94a00a44336ae314deceec/x/xion/keeper/grpc_query.go#L41-L65

However it's a rather ambiguous type of response.

Reply:
Kind of a symptom of SDK boilerplate. We need to define the specific return type of this query, but really all we care about is "did the query error". If the query didn't error, we expect the WebAuthN authentication was valid. 

This call is currently only used in the AA contract for WebAuthN verification